tfhe/src/high_level_api/mod.rs

```diff
@@ -129,35 +130,37 @@ pub enum Device {
 }
 
 #[derive(Copy, Clone, PartialEq, Eq, Debug)]
+#[repr(i32)]
+#[derive(EnumIter)]
```

Comment:
derive only in #[cfg(test)] maybe ? avoids having another mandatory dependency

so #[cfg_attr(test, derive(EnumIter))] I believe

Reply:
Done